routes/purchasesOrders.js

```diff
@@ -31,17 +38,17 @@ router.get('/:id', (req, res) => {
 
 router.post('/', (req, res) => {
   const formData = req.body;
-  if (formData.title === null) {
+  if (formData.DVM_id === "" || formData.quantity === "") {
```

Comment:
you test if the fields are empty, but not if they are present. that was the purpose of the "== null": you need both tests ;)